mercury/protocol-issue-credential/src/main/scala/org/hyperledger/identus/mercury/protocol/issuecredential/RequestCredential.scala

```diff
@@ -67,7 +67,7 @@ object RequestCredential {
       body = RequestCredential.Body(goal_code = oc.body.goal_code, comment = oc.body.comment),
       attachments = oc.attachments,
       thid = oc.thid.orElse(Some(oc.id)),
-      from = oc.to,
+      from = oc.to.getOrElse(throw new IllegalArgumentException("OfferCredential must have a recipient")),
```

Comment:
Same here. But you need to change the method signature to return a ZIO that may fail

Reply:
This function in never used any where ,  I dont want to delete it , I think it will be used if we decide Issuecredntail start with Holder for future